Added corrected work on Hypno and Knut without scene

JeremyAstolfi opened the pull request.

liam-middlebrook commented:
It looks like you deleted most of the project with this commit. Are you sure this is what you wanted to do?

Commit:
Added in the class prototype scene. Also changed

a few aspects of how the Wind functions (the force
is now input as a full Vector2 instead of just a
scalar). And put in a script only for this prototype
for a fun reward for beating the level.

liam-middlebrook committed:
Fix TimeController Listener Lifetimes

Commit:
This adds in one of my scenes, but more importantly

it adds prefabs for a spawner, standardizes the player
prefab as the one we used in the last prototype, and
also adds the spawner, the player, and a new underground
ground tile to the tile editor.

Sam-Handrick committed:
Finished implemneting the new spawner system.

Sam-Handrick committed:
Fixed a problem with the previous push; now all the

new prefabs should be in.

Sam-Handrick committed:
One more final fix due to a null prefab reference.

JeremyAstolfi committed:
Added the assets to the updated master

JeremyAstolfi committed:
Trying it again

JeremyAstolfi committed:
Trying enemy asset commit again and again

JeremyAstolfi committed:
Everything is fixed
